.travis.yml

```diff
@@ -40,6 +40,9 @@ matrix:
     - php: 7.1
       env:
         - SYMFONY_VERSION='3.2.*'
```

Comment:
symfony 3.1 and 3.2 are outdated already and non-lts. so i'd say we don't need to tests against those anymore but instead 3.4 which is lts

Reply:
I added 3.4 to the tests (as default for all php versions)
3.2 is officially still security supported for a month. But yes, we could remove those two.